draft-iab-protocol-maintenance.md

```diff
@@ -113,6 +113,25 @@ and implementations being strict in following the requirements of
 those specifications.
 
 
+## Applicability
+
+The guidance in this document is intended for protocols that are deployed to the
+Internet. There are some situations in which this guidance might not apply to a
+protocol due to conditions on its implementation or deployment.
+
+In particular, this guidance depends on an ability to ongoingly update and
+deploy implementations. Being able to update implementations that are deployed
+to the Internet is an essential part of managing security risk. In practice, not
+all protocol deployments are continuously maintained.
+
+Where implementations are not updated, there is no opportunity to apply the
+practices that this document recommends. In particular, some practices - such as
+those described in {{intolerance}} - only exist to support the development of
+protocol maintenance and evolution. Employing this guidance is therefore only
+applicable where there is the possibility of improving deployments through
+updates of their implementations.
+
```

Comment:
I wanted to leave one more question here.  We acknowledge the need to add some temporary hacks when updates are not possible in the short term.  Is this a good place to place a signpost for that possibility?